packages/connector/interface/src/entities/ProjectConfig.kt

```diff
@@ -0,0 +1,54 @@
+package entities
```

Comment:
```suggestion
package io.tellery.entities
```
I'm guessing that the best practice in the kotlin style guide might not be followed by the autocomplete of idea.
maybe we should fall back to package-based folder structure ?

Reply:
sure, it is more convenient.